src/nodedef.h

```diff
@@ -382,6 +382,8 @@ class IWritableNodeDefManager : public INodeDefManager {
 			const ContentFeatures &def)=0;
 	// If returns CONTENT_IGNORE, could not allocate id
 	virtual content_t allocateDummy(const std::string &name)=0;
+	// Unallocate an id
+	virtual void unallocateName(const std::string &name)=0;
```

Comment:
It's deallocate.  Deallocate, or free.
Not un-allocate.  This is grammatically incorrect English.